app/helpers/handlers.js

```diff
@@ -165,6 +165,9 @@ const getPage = async (question, request, h) => {
         }
       }
       break
+      case 'potential-amount': 
+        question.maybeEligibleContent.isconcreteBunkerSize = Number(getYarValue(request, 'concreteBunkerSize')) >= 100 ? true : false
```

Comment:
Great logic, but we only show the extra content if concreteBunkerSize is 101 or more

aka should be concreteBunkerSize > 100

also, we would need to update the email/dora code, so if concreteBunkerSize >= 100, we set it as 100

Reply:
There won't be any issue if we show concrete bunker sizes over 100 because the only affected thing is the total requested grant amount.